onnxruntime/core/providers/cuda/cuda_provider_options.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+namespace onnxruntime {
+
+/**
+  * Configuration information for a cuda provider.
```

Comment:
Would be better if you can leave a comment here to say: this is somewhat internal, not accessible from the public C/C++/C#/... APIs except the public python API.

Reply:
I've added the comment.